packages/api/tests/api/users.test.ts

```diff
@@ -14,7 +14,66 @@ const sampleUser: Partial<User> = {
 
 const loggerSpy = jest.spyOn(logger, 'error').mockImplementation();
 
-describe('users API GET route', () => {
+describe('users API GET route for currently logged in user', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+  it('401 error when not logged in', async () => {
+    const handler = testHandler(users);
+
+    const { text } = await handler.get('/me').expect(401);
+    expect(text).toEqual('You must be logged in.');
+    expect(handler.entityManager.findOne).toBeCalledTimes(0);
+  });
+
+  it('404 error current user not found in database', async () => {
+    const handler = testHandler(users, (req, _res, next) => {
+      req.user = { githubToken: 'abcd123', profile: { id: 'aaa' } };
+      next();
+    });
+
+    handler.entityManager.findOne.mockResolvedValueOnce(null);
+
+    await handler.get('/me').expect(404);
+    expect(handler.entityManager.findOne).toBeCalledTimes(1);
+    expect(handler.entityManager.findOne).toHaveBeenCalledWith(User, { githubId: 'aaa' });
```

Comment:
Can you move the `githubId` out to a constant for both uses here?

Move `aaa` out to a constant (within individual tests -not a shared one for the file) for all relevant tests - whenever you set a mock value and then use that value in an `expect`, you should always reference a shared variable, not the raw value (that way if you change one, your tests don't fail)